modules/web/src/app/routing.ts

```diff
@@ -62,6 +67,10 @@ function createRouting(): Routes {
           path: 'projects/:projectID/clusters',
           loadChildren: () => import('./cluster/module').then(m => m.ClusterModule),
         },
+        {
+          path: 'projects/:projectID/external',
```

Comment:
As Above of one of my comment changing from `external` will required update here too
```suggestion
          path: 'projects/:projectID/externalclusters', 
```